.github/workflows/windows.yml

```diff
@@ -2,18 +2,13 @@ name: windows
 
 on: [push, pull_request]
 
-env:
-  NUGET_URL: https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe
-
 jobs:
   windows_build:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v1
-    - name: Download Nuget.exe
-      run: Invoke-WebRequest -OutFile Nuget.exe $Env:NUGET_URL
     - name: Install Dependencies
-      run: ./Nuget.exe restore App8\App8.sln
+      run: dotnet restore App8\App8.sln
```

Comment:
cool, then  if this still passes on CI (I still dont know cause I dont see the green tick) then we can replace `msbuild` with `dotnet build` too?